src/ConnectedMode/Migration/Wizard/MigrationWizardWindow.xaml

```diff
@@ -111,6 +111,16 @@
                     </TextBlock>
                 </Border>
 
+                <CheckBox x:Name="chk_SaveSharedBinding">
```

Comment:
This PR doesn't have any checkbox logic. Will you add that later or?

Reply:
yes